wooey/models/core.py

```diff
@@ -108,6 +108,13 @@ class ScriptVersion(ModelDiffMixin, WooeyPy2Mixin, models.Model):
     created_date = models.DateTimeField(auto_now_add=True)
     modified_date = models.DateTimeField(auto_now=True)
 
+    created_by = models.ForeignKey(
+        User, related_name='created_script_version', on_delete=models.SET_NULL, null=True, blank=True
```

Comment:
This should be plural, can we keep the convention of `set` and make these `created_script_version_set`?